aws/request/request_retry_test.go

```diff
@@ -0,0 +1,107 @@
+package request
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+)
+
+func r(t *testing.T, url string) *http.Request {
```

Comment:
Would prefer the `r` function be renamed to `newRequest` to prevent confusion/shadowing with the `r` variables below.

Reply:
done